plugins/auto-sizes/tests/test-improve-sizes.php

```diff
@@ -31,12 +31,13 @@ public function set_up(): void {
 
 		// Disable auto sizes.
 		remove_filter( 'wp_content_img_tag', 'auto_sizes_update_content_img_tag' );
+
+		// Disable lazy loading attribute.
+		add_filter( 'wp_img_tag_add_loading_attr', '__return_false' );
```

Comment:
Why does this filter need to be disabled for these tests?

Reply:
Due to the `auto` value being added to the `sizes` attribute through WordPress Core, if we cannot disable the filter, I will need to update the majority of the tests.  

For example, the current assertion:  
```php
$this->assertStringContainsString( 'sizes="100vw" ', $result );
```  
would need to be replaced with:  
```php
$this->assertStringContainsString( 'sizes="auto, 100vw" ', $result );
```